### PR TITLE
4597 doclet update

### LINKED ISSUE #4597: Reinstate commented out blocks for Java11
To get Java11 to build, these elements are not being built/performed:

* javadoc for ease_scripting (steps in `org.mwc.debrief.help/collate_docbook.xml`)
* `org.mwc.debrief.UI_Test`

### PR DESCRIPTION
**Pull request by saulhidalgoaular**
Fix #4597

### CODE REVIEW

File: org.mwc.debrief.help/collate_docbook.xml [hunk 1]
```diff
@@ -333,18 +328,19 @@
             <arg value="-failOnHTMLError"/>
             <arg value="false"/>
             <arg value="-subpackages"/>
-            <arg value="."/>
+            <arg value="org.mwc.debrief.scripting.wrappers"/>
+            <arg value="-link"/>
+        	<arg value="https://docs.oracle.com/javase/8/docs/api"/>
```
Comment (IanMayo) [inline org.mwc.debrief.help/collate_docbook.xml]: @saulhidalgoaular - we are building on Java11 now, does this need to be updated?

Comment (saulhidalgoaular) [reply org.mwc.debrief.help/collate_docbook.xml]: Fixed :)

Comment (saulhidalgoaular) [reply org.mwc.debrief.help/collate_docbook.xml]: I have used the offline option, because I didn't find the `package-list` online.

**Comment by saulhidalgoaular** [comment]
I conclude This fix is not needed anymore.

**Pull request closed without merging**